### PR TITLE
Fix Gulp configuration

### PR DESCRIPTION
**Pull request by senioroman4uk**
Configured gulp to use relative path for injection of dependencies. Now before project building project Visual Studio runs `gulp serve-dev --vs` 
This will update the index.html in the root of entry-point.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/20%23discussion_r58486818%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/20%23discussion_r58486903%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20e654fc9da25a0fedfc53cdecf70d966851757a1c%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html%2012%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/20%23discussion_r58486818%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22we%20can%20just%20skip%20these%20weird%20changes%20%3A%29%22%2C%20%22created_at%22%3A%20%222016-04-05T04%3A55%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html%3AL37-47%22%7D%2C%20%22Pull%20e654fc9da25a0fedfc53cdecf70d966851757a1c%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html%2014%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/20%23discussion_r58486903%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%60../../%60%20is%20correct%3F%22%2C%20%22created_at%22%3A%20%222016-04-05T04%3A57%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html%3AL14-34%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull e654fc9da25a0fedfc53cdecf70d966851757a1c DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html 12'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/20#discussion_r58486818'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html:L37-47</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> we can just skip these weird changes :)
- [x] <a href='#crh-comment-Pull e654fc9da25a0fedfc53cdecf70d966851757a1c DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html 14'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/20#discussion_r58486903'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html:L14-34</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> `../../` is correct?

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/20?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/20?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-013/pull/20'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ITsvetkoFF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ITsvetkoFF/Kv-013/pull/20'>Refresh</a>.<a href='#crh-update-none'></a>

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html [hunk 1]
```diff
@@ -37,6 +37,10 @@
...
             </div>
+            <div class="row" ng-include="">
+
+            </div>
         </div>
     </section>
-</section>
\ No newline at end of file
+</section>
+qweryyu
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html]: we can just skip these weird changes :)

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html [hunk 2]
```diff
@@ -14,20 +14,20 @@
...
-    <link rel="stylesheet" href="/bower_components/toastr/toastr.css" />
+    <link rel="stylesheet" href="./Frontend/bower_components/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="./Frontend/bower_components/toastr/toastr.css" />
     <!-- endbower -->
     <!-- endbuild -->
 
     <!-- build:css styles/app.css -->
     <!-- inject:css -->
-    <link rel="stylesheet" href="/.tmp/styles.css">
+    <link rel="stylesheet" href="./FrontEnd/src/client/../../.tmp/styles.css">
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html:24]: `../../` is correct?

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html (hunk 1 above)
Comment (senioroman4uk) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/dashboard/dashboard.html]: include will be related for activity ui, cleared.
Second one is my bad, missed that, cleared.

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html (hunk 2 above)
Comment (senioroman4uk) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/index.html:24]: Yes it's ok, this path is being generated by gulp inject

**Pull request merged**